src/main/java/se/lth/base/server/mail/CustomEmail.java

```diff
@@ -2,64 +2,510 @@
 
 import org.simplejavamail.email.Email;
 import org.simplejavamail.email.EmailBuilder;
+import org.simplejavamail.email.Recipient;
+import org.slf4j.LoggerFactory;
+import se.lth.base.server.Config;
+import se.lth.base.server.data.*;
+
+import javax.mail.Message;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
 
 /**
- * Creates a SimpleEmail object based on a standard predefined email type
- * 
- * @author Group 1 ETSN05 2018
+ * Creates a SimpleEmail object based on a standard predefined EmailType
  *
+ * @author Group 1 ETSN05 2018
  */
 public class CustomEmail {
-	public static final int WELCOME_EMAIL = 1;
-	
-	private String firstName;
-	private String lastName;
-	private String email;
-	private int emailType;
-
-	public CustomEmail(String firstName, String lastName, String email, int emailType) {
-		this.firstName = firstName;
-		this.lastName = lastName;
-		this.email = email;
-		this.emailType = emailType;
-	}
-	
-	public Email getEmail() {
-		return EmailBuilder.startingBlank()
-				.from(MailHandler.DEFAULT_SENDER)
-				.to(firstName + " " + lastName, email)
-				.withSubject(getSubject())
-				.withHTMLText(getMessage())
-		        .withPlainText("Please view this email in a modern email client!")
-		        .buildEmail(); 	
-	}
-	
-	public String getSubject() {		
-		switch (emailType) {
-		case WELCOME_EMAIL: return  getWelcomeSubject();
-		default: return "INVALID EMAIL TYPE";
-		}
-	}
-	
-	public String getMessage() {
-		switch (emailType) {
-		case WELCOME_EMAIL: return  getWelcomeMessage();
-		default: return "INVALID EMAIL TYPE";
-		}
-	}
-	
-	// ---------- STANDARD EMAILS BELOW --------------
-	
-	private String getWelcomeSubject() {
-		return "Welcome to CommuteCompanion " + firstName;
-	}
-	
-	private String getWelcomeMessage() {
-		StringBuilder sb = new StringBuilder();
-		
-		sb.append("<h1>Hi ").append(firstName).append("</h1>");
-		sb.append("<p>Welcome to CommuteCompanion!</p>");
-		
-		return sb.toString();
-	}
+    private final static String TITLE = "title", PREVIEW = "preview", INTRO = "intro",
+            BUTTON = "button", OUTRO = "outro", WEBSITE_LINK = "http://www.yourcommutecompanion.herokuapp.com",
+            EMAIL_TEMPLATE = "email-template.html";
```

Comment:
Should avoid single line declarations like this. It makes debugging more difiicult and worsens the readability.